### PR TITLE
feat : Homepage UI

### PR DESCRIPTION
**Pull request by Priya-Patel31**
* Added backend 
* Improved homepage UI .

You can omit reviewing the CSS file as it has been already reviewed .

### CODE REVIEW

File: src/backend/controllers/QuizController.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  try {
+    const quizId = request.params.quizId;
+    const quiz = schema.quizzes.findBy({ _id: quizId }).attrs;
+    return new Response(200, {}, { quiz });
+  } catch (error) {
+    console.log(error);
+    return new Response(
+      404,
+      {},
+      { errors: ["The user you request does not exist. Not Found error."] }
```
Comment (Ankur9669) [inline src/backend/controllers/QuizController.js:13]: Suggestion: You send a different error here

**Review by Ankur9669 — APPROVED** [review]
All the files are looking good to me.

File: src/pages/home/home.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    border-radius:2rem;
+  }
+  .active{
+    background-color: var(--accent-color);
+    border-radius: 2rem;
+  
+  }
+  .logout {
+    position: absolute;
+    bottom: 32px;
```
Comment (t007rushi) [inline src/pages/home/home.css:31]: 2rem as per your convention

File: src/services/homePage_services.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import data from "../utils/trophy.json"
+
+export const getCategories = ()=>{
+    console.log(data)
```
Comment (t007rushi) [inline src/services/homePage_services.js:4]: console here

**Review by t007rushi — APPROVED** [review]
looks great UI 
neat and clean file structure 
keep it up and go ahead

File: src/pages/home/home.css (hunk 2 above)
Comment (Priya-Patel31) [reply src/pages/home/home.css:31]: I'll change it.

**Review by krate27 — APPROVED** [review]
Things look good. One thing which could be done is use of good css formatter.
Rest is good to merge.

**Pull request merged**